### ISSUE TITLE
2.7.3 版本支持

### ISSUE THREAD

**Issue by phoenixg** [NoBey/runjs-crack#3]
执行 runjs-crack
修改成功, 重启后随意激活

但还是有 activate 阻拦，未成功

```
{
  "version": "2.7.3",
  "platform": "darwin",
  "arch": "arm64",
  "activated": false,
  "logs": []
}
```

**Comment by konstantinov-it** [comment]
Hi! This is very relevant. I would be grateful for a software update for the current version of runjs

**Comment by optionsx** [comment]
yes it doesn't work... i modified the code to work on windows but to no avail
removing their condition checks should be easy..

**Comment by asmirbelkic** [comment]
> 

Can you provide a PR

**Comment by Pab450** [comment]
If you're still interested you can find a cracking tool for version 2.10.1 here: 
[RunJS Cracker](https://github.com/Pab450/RunJS-Cracker)

**Comment by Ronald3217** [comment]
> If you're still interested you can find a cracking tool for version 2.9.0 here: [RunJS Cracker](https://github.com/Pab450/RunJS-Cracker)

Thank you very much, I had version 2.9.0 cracked after a few hours reviewing the code of this repo and rebuilding it, but it was a crude way of doing it, this repo of yours helped me a lot in addition to making it easier for me not to have to decompress my . zip every time I need it.

Thank you very much, and I hope (if you can) you keep the repository updated for new versions.

**Comment by Pab450** [comment]
> > If you're still interested you can find a cracking tool for version 2.9.0 here: [RunJS Cracker](https://github.com/Pab450/RunJS-Cracker)
> 
> Thank you very much, I had version 2.9.0 cracked after a few hours reviewing the code of this repo and rebuilding it, but it was a crude way of doing it, this repo of yours helped me a lot in addition to making it easier for me not to have to decompress my . zip every time I need it.
> 
> Thank you very much, and I hope (if you can) you keep the repository updated for new versions.

Glad I could help! I'll do my best to keep the repo up to date, so don't hesitate to open an issue when a new version comes out!

**Comment by amartinezdev** [comment]
@Pab450 hey dude, I asked you something on reddit, can you check it please? It's about RunJS
